tests/zfs-tests/tests/functional/pool_checkpoint/setup.ksh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/ksh -p
```

Comment:
This should be `#!/bin/ksh` for all the new test scripts.  It was causing the CentOS 6 failures.

Reply:
Yes!  I saw that in one of the tests introduced by vdev removal and totally forgot to look for it.